terminus/models/city.py

```diff
@@ -50,12 +50,12 @@ def add_intersection_at(self, point):
                     road.replace_node_at(point, intersection)
 
     def bounding_box(self):
-        child_bounding_boxes = [self._bounding_box_from_list(self.roads),
```

Comment:
Because if the city didn't have any blocks, or buildings, or roads, it didn't work because it attempted to create a bounding box from an empty list (it happened with one of the samples). Now, as long as the city has at least one thing, it works well because it makes a list of ALL the bounding boxes before using from_boxes. If the city is not empty, there is at least one box in that list.